keps/sig-network/2091-cluster-network-policy/README.md

```diff
@@ -0,0 +1,1161 @@
+the feature may cause created cluster-scoped policy resources to be deleted,
+which may impact the security of the cluster. Administrators must make provision
+to secure their cluster by other means before disabling the feature gate.
+
+###### What happens if we reenable the feature if it was previously rolled back?
+
+n/a
+
+###### Are there any tests for feature enablement/disablement?
+
+n/a
+
+### Scalability
+
+<!--
+For alpha, this section is encouraged: reviewers should consider these questions
+and attempt to answer them.
+
+For beta, this section is required: reviewers must answer these questions.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+-->
+
+###### Will enabling / using this feature result in any new API calls?
+
+Enabling this feature by itself will have no impact on the cluster and no
+new API calls will be made.
+
+###### Will enabling / using this feature result in introducing new API types?
+
+Enabling this feature will introduce new API types as described in the [design](#design-details)
+section. The supported number of objects per cluster will depend on the individual
+CNI providers who will be responsible to provide the implementation to realize
+these resources.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+n/a
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+n/a
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+n/a
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+n/a
+
+## Implementation History
+
+- 2021-02-18 - Created initial PR for the KEP
+
+## Drawbacks
+
+Securing traffic for a cluster for administrator's use case can get complex.
+This leads to introduction of a more complex set of APIs which could confuse
+users.
+
+## Alternatives
+
+Following alternative approaches were considered:
+
+### NetworkPolicy v2
+
+A new version for NetworkPolicy, v2, was evaluated to address features and use cases
+documented in this KEP. Since the NetworkPolicy resource already exists, it would be
+a low barrier to entry and can be extended to incorporate admin use cases.
+However, this idea was rejected because the NetworkPolicy resource was introduced
+solely to satisfy a developers intent. Thus, adding new use cases for a cluster admin
+would be contradictory. In addition to that, the administrator use cases are mainly
+scoped to the cluster as opposed to the NetworkPolicy resource, which is `namespaced`.
+
+### Empower, Deny, Allow action based CRD
+
+Unlike the NetworkPolicy resource in which each rule represents an allowed
+traffic, ClusterNetworkPolicy will enable administrators to set `Empower`,
```

Comment:
uh... no it won't...

This needs to be rewritten to reflect the fact that we _aren't_ implementing it this way

Reply:
We wanted to mention these in the `Alternatives considered` section just to show it to the readers that, we have explored these other options but did not work out / was opposed by the community. I can try to make this section more concise, or simply redact it if you do think it's not needed at all.

Reply:
Listing the old alternative ideas is good, but the text should be rewritten to talk about these approaches as things we _aren't_ doing (and explaining why we decided against them), rather than just preserving the old text verbatim and making it sound like we _are_ doing these things.

Reply:
Done